xonsh/tokenize.py

```diff
@@ -113,18 +110,16 @@
     "DOLLARNAME",
     "IOREDIRECT",
 ]
-HAS_ASYNC = (3, 5, 0) <= PYTHON_VERSION_INFO < (3, 7, 0)
+HAS_ASYNC = PYTHON_VERSION_INFO < (3, 7, 0)
```

Comment:
I think we can remove this check (and the conditionals below it) entirely.  All supported version of Python should be able to handle having `ASYNC` and `AWAIT` tokens now.

Reply:
I _think_ that this is because ASYNC/AWAIT are _soft_ keyword below 3.7 and that will make some technically 3.7 compatible code broken. But we can _try_

Reply:
Nop. Does not work.